### PR TITLE
manage all .yml files

### PR DESCRIPTION
**Pull request by YUzhva**
### JIRA:
**LR-165 Remove all unnecessary yml files**: https://ssu-jira.softserveinc.com/browse/LR-165
as subtask of main task: **LR-167 Cleanup **: https://ssu-jira.softserveinc.com/browse/LR-167

### Author.

- [x] I have added link to this PR to the JIRA
- [x] I have logged time in JIRA.

### Reviewers.

> @brouhaha88
> @sergiy17
> @superNazo

- [x] @anatoliiibrahimov
- [x] @OKrain
- [x] @vovka

### Issue summary.
Secure all gently information from pushing to our repo.

### Fix summary. 
1) All files with extention .yml was renamed to the .yml.example
2) Setup codeship and wrote script for heroku to rename all .yml.example to .yml files
3) + bugfix: coveralls repo_token moved into application.yml.example
_Short description of how you implemented a feature or fixed a bug. And why._

### Testing approach.
Manual testing (rails s locally  + codeship as CI and heroku for production in special mytmpapp) application still works :fire:

### CODE REVIEW

File: .profile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# The .profile script will be sourced after the app’s config vars at Heroku.
+cp config/database.yml.example config/database.yml
```
Comment (vovka) [inline .profile]: Do we really need to copt database yml on heroku?

**Comment by vovka** [comment]
Since we use figaro to manage env vars then i don't think we have to hide secrets.yml because it doesn't contain any sensitive information.

**Comment by vovka** [comment]
What about `/test` directory?

**Comment by YUzhva** [comment]
<<  i don't think we have to hide secrets.yml because it doesn't contain any sensitive information.

Yeah, but .gitignore says:
+/config/*.yml

or we could change gitignore to:
+/config/application.yml
+/config/database.yml

**Comment by vovka** [comment]
We can keep *.yml in gitignore and add force secrets.yml

**Comment by vovka** [comment]
@YUzhva see comments ^^^^^

**Comment by vovka** [comment]
:+1:

**Pull request merged**